defaults/main.yml

```diff
@@ -48,7 +48,7 @@ grsecurity_build_revision: "{{ linux_kernel_version }}-grsec-{{ grsecurity_versi
 
 # The command to run to perform the compilation. Does not include environment
 # variables, such as PATH munging for ccache and setting workers to number of VCPUs.
-grsecurity_build_compile_command: make deb-pkg
+grsecurity_build_compile_command: "make -j${CONCURRENCY_LEVEL} deb-pkg"
```

Comment:
This change is not as it appears. First, it won't set the jobs to the number of cores, since we use `command` and not `shell` for the compile command. From `ansible-doc command`:

>  The `command` module takes the command name followed by a list of space-delimited arguments. The given command will be executed on all selected nodes. It will not be processed through the shell, so variables like `$HOME` and operations like `"<"`, `">"`, `"|"`, `";"` and `"&"` will not work (use the [shell] module if you need these features). For Windows targets, use the [win_command] module instead.

Was still able to complete a build with the change, so it doesn't _break_ the build, but it's actually running `make -j deb-pkg`, which... might be surprising. From `make -h`:

> -j [N], --jobs[=N]          Allow N jobs at once; infinite jobs with no arg.

So... that doesn't seem like a sane default to me. (Might explain why system load hit 10 when I tried this.) How about we:

* Create a default role var called `grsecurity_build_compile_jobs`
* Set the new var value to `{{ ansible_processor_vcpus }}`
* Delete the `CONCURRENCY_LEVEL` from the `grsecurity_build_env_vars` default var (since it's only relevant to `make-kpkg`)

That should provide a sane default and also a convenient method for overrides.

Reply:
❤️  This is awesome feedback and review @conorsch !! thanks buddeh. 

@emkll - this was something i tacked on to your branch but really its kind of unrelated to your PR. So i'll rebase this branch while dropping this commit and make a new PR just for this commit.